manuscripts2/metrics/git.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+#
+# Copyright (C) 2018 CHAOSS
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Author:
+#   Pranjal Aswani <aswani.pranjal@gmail.com>
+#
+
+import sys
+sys.path.insert(0, '..')
+
+from manuscripts2.new_functions import Query
+
+
+class GitMetrics():
+
+    def __init__(self, index):
+
+        self.name = "git"
+        self.commits = Query(index)
+
+    def get_section_metrics(self):
+
+        return {
```

Comment:
This is very much JavaScript-like ;-) I'm not completely sure yet, but I think we could improve it. But for now, that's ok with me.

Reply:
I am sorry, I don't get the reference here. What is JS like?